common/src/jni/main/cpp/NativeCrypto.cpp

```diff
@@ -7622,8 +7622,6 @@ static void NativeCrypto_SSL_shutdown(JNIEnv* env, jclass, jlong ssl_address,
         return;
     }
     if (fdObject == nullptr) {
-        Errors::jniThrowNullPointerException(env, "fd == null");
-        JNI_TRACE("ssl=%p NativeCrypto_SSL_shutdown => fd == null", ssl);
```

Comment:
yeah, fixes some of the tests that rely on being able to shutdown without ever having been connected. OpenJDK lazy creates the fd.